build_dependencies.sh

```diff
@@ -7,13 +7,21 @@ buildDIDComm() {
     cd $DIDCommFolder/wasm
     wasm-pack build --target=web --out-dir=../../../generated/didcomm-wasm-browser
     wasm-pack build --target=nodejs --out-dir=../../../generated/didcomm-wasm-node
+    #TODO: find better way to approach this
+    #This code fails on browser when wasm is first loaded, it can just be ignored
+    #The code will fully work
+    sed "/if (typeof input === 'undefined') {/,/}/d" ../../../generated/didcomm-wasm-browser/didcomm_js.js > temp_file && mv temp_file ../../../generated/didcomm-wasm-browser/didcomm_js.js
```

Comment:
not an issue, just an idea so can be done later, but this editing of the code here and below should probably be handled by rollup?

Reply:
It could potentially be done in Rollup, correct.